src/bundle/Controller/Permission/LanguageLimitationController.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\AdminUi\Controller\Permission;
+
+use Ibexa\AdminUi\Permission\LimitationResolverInterface;
+use Ibexa\Contracts\AdminUi\Controller\Controller;
+use Ibexa\Contracts\Core\Repository\Values\Content\Content;
+use Ibexa\Contracts\Core\Repository\Values\Content\Location;
+use Ibexa\Contracts\Core\Repository\Values\Content\VersionInfo;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Response;
+
+final class LanguageLimitationController extends Controller
+{
+    private LimitationResolverInterface $limitationResolver;
+
+    public function __construct(LimitationResolverInterface $limitationResolver)
+    {
+        $this->limitationResolver = $limitationResolver;
+    }
+
+    public function loadLanguageLimitationsForContentAction(
+        Content $content,
+        ?VersionInfo $versionInfo = null,
+        ?Location $location = null
+    ): Response {
+        return new JsonResponse(
+            $this->limitationResolver->getLanguageLimitations(
+                $versionInfo ?? $content->getVersionInfo(),
+                $location ?? $content->getVersionInfo()->getContentInfo()->getMainLocation()
+            )
+        );
+    }
```

Comment:
Given the nature of this response I feel like it should be done via REST, but I don't remember if there was any discussion about it.

Reply:
It was mentioned on one of our daily meetings that decision was made to add this as "regular" action.